test/unit/core/dom/test-dom.js

```diff
@@ -505,6 +506,39 @@ describes.sandboxed('DOM', {}, (env) => {
     });
   });
 
+  describe('realChildElements and realChildNodes', () => {
```

Comment:
Let's move this into `test/unit/core/dom/test-query.js`

Reply:
Done